kieserver/modules/kieserver/install

```diff
@@ -25,6 +25,8 @@ if [[ -f "${SOURCES_DIR}/${BUSINESS_CENTRAL_DISTRIBUTION_ZIP}" ]] && [[ "${JBPM_
     fi
 fi
 
+mv /tmp/artifacts/jbpm-event-emitters-kafka-7.x.Final.jar "/opt"
```

Comment:
please use a more specific dir as I mentioned you on gchat.

Reply:
did you test cekit _dest_   parameters instead? https://docs.cekit.io/en/latest/descriptor/image.html#artifact-types

Reply:
Applied